.github/workflows/pull-request.yml

```diff
@@ -17,6 +31,7 @@ jobs:
         with:
           distribution: "zulu"
           java-version: "11"
+          cache: 'gradle'
```

Comment:
Bit of a nitpick but I notice that here and in 2 other places where you add the `cache: 'gradle'` line, you're using single quotes around `gradle` while every other quoted string in the files use double quotes... We should change for consistency unless single quote is required.